wordpress-mobile/WordPress-Android
# Site creation rework: site creation epilogue

Fixes #6930 

Modifies the "Creating" screen to mutate into the "Epilogue" screen when the process concludes. Reason for the mutation (instead of a new Fragment/screen) is to be able to use a hidden WebView preloading the newly created site at the end of the process.

The WebView is allowed 3seconds to finish loading the site until a generic "Congrats" screen is shown instead (as per the ticket).

Note: The "CONFIGURE" and "WRITE FIRST POST" buttons will be implemented in a subsequent PR.

To test:
1. With the app logged in, head over to MySite, tap on the "Switch Site" button, the "+" action on the toolbar and select "Creation WordPress.com site" option from the popup
2. Tap on any of the category cards, then on any of the themes, then input a site title in the first input box, hit "NEXT", select a domain from the list and hit "CREATE SITE"
3. The "Creating" screen will appear and the process will commence and continue
4. In the end, the new site's homepage should appear or the generic "Congrats" screen if the site took more than 3 seconds to completely load.
5. If the WebView is shown, try tapping on any of the links on the site. None should work and a "Links are disabled on the preview screen" toast should appear.
6. Hit the (hardware) Back button and notice that the app lands on the MySite screen
